spec/parsers/mp3_parser_spec.rb

```diff
@@ -144,6 +144,14 @@
     }.to raise_error(FormatParser::IOUtils::InvalidRead)
   end
 
+  it 'supports id3 v2.4.x' do
+    fpath = fixtures_dir + '/MP3/id3v24.mp3'
```

Comment:
What file is this one? If it's the one I sent you, that's private 🙂

Reply:
no no! 😄 

it's one that I created by myself!